src/Apache.IoTDB/Rpc/Generated/TConfigNodeLocation.cs

```diff
@@ -1,5 +1,5 @@
 /**
- * Autogenerated by Thrift Compiler (0.14.2)
+ * Autogenerated by Thrift Compiler (0.13.0)
```

Comment:
IoTDB java client is using Thrift Compiler 0.14.1. Should we using the same version?

Reply:
I think IoTDB itself is abled to generate csharp code of Thrift. Just adding a dotnet label like this.

https://github.com/apache/iotdb/blob/0c66c2f1c92de6a841e542d17fb7a16c99be914f/pom.xml#L1640-L1651

Reply:
Agreed. I'll upgrade the Thrift Compiler to version 0.14.1 accordingly.